cli_curr/admin.go

```diff
@@ -560,6 +500,21 @@ func newAdminClusterCommands() []cli.Command {
 				AdminDescribeCluster(c)
 			},
 		},
+		{
+			Name:    "list",
+			Aliases: []string{"ls"},
+			Usage:   "List clusters information",
+			Flags: []cli.Flag{
+				cli.IntFlag{
+					Name:  FlagPageSize,
+					Value: 100,
+					Usage: "Page size (optional, default: 100)",
```

Comment:
```suggestion
					Usage: "Page size",
```

default message is added automatically